db/migrate/20230419084212_add_tree_nodes_overrides.rb

```diff
@@ -36,6 +39,14 @@
       LEFT JOIN goods_nomenclature_tree_node_overrides overrides ON
         indents.goods_nomenclature_indent_sid = overrides.goods_nomenclature_indent_sid
         AND indents.operation_date < coalesce(overrides.updated_at, overrides.created_at)
+      LEFT JOIN goods_nomenclatures grouping_headings ON
+        indents.goods_nomenclature_item_id NOT LIKE '%00000000'
+        AND grouping_headings.producline_suffix < '80'
```

Comment:
So from memory 100% of grouping headings are pls of 10. I guess we want to handle the future where there can be multiple grouping headings

Reply:
And maybe < is more performant on a presorted list of goods nomenclatures

Reply:
You're right they are at present and I had a debate around this one. The GoodsNomenclatureMapper has logic for multiple levels of grouping subheading - I tested the query both with on PLS=10 and the 'count grouping headings' approach and there was no performance difference so I'd opted to follow the existing code.

If they are never anything other then 10 - it'll have no impact, if they are then it'll work (I did some manual testing to make sure it behaves as expected)